src/components/App.tsx

```diff
@@ -40,17 +40,17 @@ export default function App({
   const { warning } = useSiteMetadata()
   const { accountId } = useWeb3()
   const { isInPurgatory, purgatoryData } = useAccountPurgatory(accountId)
-  const { isGraphSynced, blockHead, blockGraph } = useGraphSyncStatus()
+  // const { isGraphSynced, blockHead, blockGraph } = useGraphSyncStatus()
 
   return (
     <Styles>
       <div className={styles.app}>
-        {(props as PageProps).uri.includes('/asset') && !isGraphSynced && (
+        {/* {(props as PageProps).uri.includes('/asset') && !isGraphSynced && (
```

Comment:
so far the place we put the warning seems prominent enough so we should not need all the commented out stuff in here, so safe to just remove